testing-guidance/workflow-tests/scripting/script-python-tests.md

```diff
@@ -0,0 +1,37 @@
+# Python Scripts Workflow Tests
+
+Testing in this area should focus on the functionality of executing python scripts.
+
+## Common Issues to Watch For
+
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+1. Asset processor errors when importing/executing python scripts
+2. Warnings or errors that appear in the Editor console
+3. Unresponsive UI elements 
+
+### Platforms:
+- Windows
+- Linux
+
+### Documents and Common Terms
+[O3DE Documentation: Editor Python Bindings Gem](https://www.o3de.org/docs/user-guide/gems/reference/script/python/editor-python-bindings/) <br>
```

Comment:
Space between.

Remove the line break.

Reply:
Removed the line break, created a space between title and body.